src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
It shouldn't be called twice. One capture should emit only one event. Thanks for pointing out that single modifiers don't work. I am about to investigate it more, but I am not sure that is good idea to store the modifiers in the service.

Like I said, the bug occurs in the popover. I can capture the escape of keydown/keyup (I don't remember now which) event which should be intercepted by the capture button at the recording.

I don't understand what do you mean by "Sometimes easier is not necessary better." I believe the code should be always as simple as possible. So if more simple == easier then it is better. But I don't understand how is it connecting to this topic.

Some changes definitly should be done else the closing could not work with escape. If it needs radical changes, then that should be. (In my opinion my changes aren't radical.) I don't see how else could it be solved in a minor way. If you'll come up with a less complex one then please send a PR for it or provide suggestions for how could I reduce the complexity with providing a working solution.